sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/fluent/models/AgentPoolInner.java

```diff
@@ -1202,6 +1203,29 @@ public AgentPoolInner withNetworkProfile(AgentPoolNetworkProfile networkProfile)
         return this;
     }
 
+    /**
+     * Get the windowsProfile property: The Windows agent pool's specific profile.
+     * 
+     * @return the windowsProfile value.
+     */
+    public AgentPoolWindowsProfile windowsProfile() {
+        return this.innerProperties() == null ? null : this.innerProperties().windowsProfile();
+    }
+
+    /**
+     * Set the windowsProfile property: The Windows agent pool's specific profile.
+     * 
+     * @param windowsProfile the windowsProfile value to set.
+     * @return the AgentPoolInner object itself.
+     */
+    public AgentPoolInner withWindowsProfile(AgentPoolWindowsProfile windowsProfile) {
```

Comment:
Please add a line here https://github.com/Azure/azure-sdk-for-java/blob/00f64d851a171ec272bd56bfb466a866265a2f7b/sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterAgentPoolImpl.java#L280-L323
to set this new property.

Reply:
The change to set the new property `WindowsProfile` has been committed as a separate commit。